### ISSUE TITLE
Try not to import numpy from its source directory

### ISSUE THREAD

**Issue by michaelmarzec** [zappa/Zappa#1356]
## Context

I am running `zappa update` and for the first time receiving the following error. I have tried switching between various python and numpy/pandas versions, but both the versions that I've historically ran and newer versions are not working.

Curious if anyone else has experienced this or has any recommendations to troubleshoot this. Thanks!

```
 ImportError: Unable to import required dependencies:
numpy: Error importing numpy: you should not try to import numpy from
    its source directory; please exit the numpy source tree, and relaunch
    your python interpreter from there.
Traceback (most recent call last):
  File "/var/task/handler.py", line 657, in lambda_handler
    return LambdaHandler.lambda_handler(event, context)
  File "/var/task/handler.py", line 241, in lambda_handler
    handler = global_handler or cls()
  File "/var/task/handler.py", line 138, in __init__
    self.app_module = importlib.import_module(self.settings.APP_MODULE)
  File "/var/lang/lib/python3.9/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 850, in exec_module
  File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
  File "/tmp/age/app.py", line 17, in <module>
    import pandas as pd
```

**Comment by michaelmarzec** [comment]
This was resolved by using python 3.9.12, numpy==1.26.0, pandas=2.1.1~

**Comment by tom-at-rewbi** [comment]
i don't know that this is "completed" so much as avoided -- can we re-open this to track getting more recent versions of numpy working?
